lib/srv/desktop/rdp/rdpclient/librdprs.h

```diff
@@ -46,14 +51,9 @@ typedef enum CGOPointerWheel {
  */
 typedef struct Client Client;
 
-/**
- * CGOError is an alias for a C string pointer, for C API clarity.
- */
-typedef char *CGOError;
-
 typedef struct ClientOrError {
   struct Client *client;
-  CGOError err;
+  enum CGOErrCode err;
```

Comment:
If I'm remembering my C - Above, you typedef'd `enum CGOErrCode` to just `CGOErrCode`, so I don't think you need `enum` before it everywhere unless this is necessary for the Rust side

Reply:
This header is autogenerated by `cbindgen`.